src/wrapper.rs

```diff
@@ -505,16 +505,16 @@ struct FieldMethods {
 
 impl FieldMethods {
     fn new(ty: &Type, ident: &Ident) -> FieldMethods {
-        let mut unesc_name = ident.to_string();
-        if unesc_name.starts_with("r#") {
-            unesc_name = format!("field_{}", &unesc_name[2..]);
+        let mut unesc_base = ident.to_string();
+        if unesc_base.starts_with("r#") {
+            unesc_base = unesc_base[2..].to_owned();
```

Comment:
`trim_start`?

Reply:
`trim_start` trims whitespace, here I'm trimming the `r#`

Reply:
Ok